splash/config.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+import __builtin__
+import ast
+import ConfigParser
+import os
+
+from . import defaults
+
+
+class ConfigError(Exception):
+    pass
+
+global CONFIG_PATH
```

Comment:
this looks brittle.. This will require changing more code, but maybe we can make this parh a Settings constructor parameter, don't create a global `settings` variable at module level? We can create it in server startup code (when the path is known) and pass it down to all components instead.